tests/testthat/test-utils.R

```diff
@@ -1,6 +1,26 @@
 test_that("can evaluate expressions for benchmarking", {
+  env <- rlang::env()
   expect_equal(
-    exprs_eval("1 + 2.3"),
+    {
+      exprs_eval("x <- 1 + 2.3", env = env)
+      env$x
+    },
+    1 + 2.3
+  )
+  expect_equal(
+    {
+      exprs_eval(y <- 1 + 2.3, env = env)
+      env$y
+    },
+    1 + 2.3
+  )
+
+  expr <- quote(z <- 1 + 2.3)
```

Comment:
can you add a test for `rlang::expr()`, I think it's the tidy eval variant of quote()`. And `rlang::quo()`, it's the same but with the environment in which the expression is created in.

Reply:
will do

Reply:
though quo is a bit cumbersome as it is not a drop in replacement for `expr`